### PR TITLE
chore: README

### PR DESCRIPTION
**Pull request by vgorkavenko**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,13 +3,75 @@
...
+TBD
+
 # Development
 
 Python version: 3.11
 
 ## Setup
 
 1. Setup poetry
+```bash
```
Comment (F4ever) [inline README.md:72]: Replace with link to poetry guide

Comment (F4ever) [reply README.md:72]: + add table with variables

Comment (F4ever) [reply README.md:72]: https://github.com/lidofinance/depositor-bot#available-variables

Comment (vgorkavenko) [reply README.md]: fixed

**Review by F4ever — APPROVED** [review]

**Pull request merged**